### DISCUSSION TITLE
{admiral} error messaging updates

### DISCUSSION THREAD

**Discussion by ddsjoberg** [pharmaverse/admiral#2330] [General]
We are seeking to improve the errors messages users see returned to them in {admiral} functions. We will be adopting the {cli} package for error messaging. If you're unfamiliar with `cli::cli_abort()`, take a moment to review some slides from our bff Hadley (just the slides in this link, the other items do not pertain to admiral work https://www.danieldsjoberg.com/ARD-onboarding/05-cli-conditions.html).

General strategy:
- [ ] Create guidelines for error messaging in admiral
- [ ] Update the checking functions in {admiraldev} to use `cli::cli_abort()`
- [ ] Update error messaging in admiral

@pharmaverse/admiral please comment here if you'd like to be involved.

**Comment by bms63** [comment]
![image](https://github.com/pharmaverse/admiral/assets/10111024/84aa51c9-b39e-4c84-9ea3-6ccac229dbbd)

**Comment by bundfussr** [comment]
I'm interested as well.

**Comment by kaz462** [comment]
:raising_hand: count me in

**Comment by manciniedoardo** [comment]
Please include me in the calls!

**Comment by millerg23** [comment]
sign me up.

**Comment by sadchla-codes** [comment]
Can I please be included as well

**Comment by ddsjoberg** [comment]
Hi All, I've shared a document with you where we can add our initial thoughts. Please add to the document, then let's meet to make a plan!

**Comment by manciniedoardo** [comment]
Closing due to successful 1.1 release - well done all 😄
